config/locales/mailers/mailer.yml

```diff
@@ -19,19 +19,75 @@ en:
     get_in_touch: Get in touch
   onboarding_mailer:
     activation_email:
+      get_in_touch_html: <a href="%{contact_url}">Get in touch</a> if you have any further questions
+      header: "%{school_name} is now live on Energy Sparks"
+      join_a_webinar: Join a webinar
+      need_some_help: Need some help?
+      paragraph_1_html: You can now use Energy Sparks to begin recording your energy saving activities. Explore our pupil-lead <strong><a href="%{activity_categories_url}">activities and educational resources</a></strong>which are designed to teach your pupils about energy whilst saving money for their school.
+      paragraph_2_html: We also have a range of <strong><a href="%{intervention_type_groups_url}">energy saving actions</a></strong> that you, other members of staff and volunteers can carry out to start improving you energy usage.
+      paragraph_3_html: View your <a href="%{school_url}">school dashboard</a> for suggestions for how to get started.
+      paragraph_4: We will be in touch shortly to confirm when we have completed getting access to your energy data.
+      paragraph_5_html: For help using Energy Sparks please read our <a href="https://energysparks.uk/resources/12/download">User Guide</a>, watch our <a href="%{user_guide_videos_url}">Training Videos</a> or join one of our <a href="%{training_url}">live webinar training</a> sessions.
```

Comment:
Assume the translators be able provide the Welsh url here?

Reply:
(and the same for the other urls in the yaml?)

Reply:
hi @urbanwide the resource user guide file is English only at the moment (I think the plan is to link to the en and cy ones in drive directly) so we can keep this link as is for the time being.  however, we do have a `user_guide_download_path` method for that specific file so will push an update so that's used here instead.

Reply:
OK thanks for clarifying 👍